src/main/java/com/teamcautionrobotics/robot/AimingLights.java

```diff
@@ -0,0 +1,24 @@
+
+package com.teamcautionrobotics.robot;
+
+import edu.wpi.first.wpilibj.*;
```

Comment:
Can we name the method for this class  "toggle" instead of "set," since that is what it is doing?

Reply:
I think set is a better name because when you call the method, it does not necessarily change state, e.g. calling aimingLights.set(true) when the lights are already on.